src/main_app/jobs_workers/public_jobs_workers/copy_svg_langs_per_file/worker.py

```
@@ -0,0 +1,462 @@
+        job_id=job_id,
+        user=user,
+        cancel_event=cancel_event,
+        args=args,
+    )
+    worker.run()
+
+
+__all__ = [
+    "copy_svg_langs_per_file_worker_entry",
+    "CopySvgLangsWorker",
+]
```

Ruff (0.15.17)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)